packages/appium/lib/utils.js

```diff
@@ -280,7 +280,7 @@ function adjustNodePath() {
  * The capabilities argument itself gets mutated, so it does not contain parsed
  * settings anymore to avoid further parsing issues.
  * Check
- * https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/settings.md
+ * https://appium.io/docs/en/2.0/guides/settings/
```

Comment:
For future compatibility it may be better to change this to `https://appium.io/docs/en/latest/guides/settings/`

Reply:
@eglitise The url as mentioned, `https://appium.io/docs/en/latest/guides/settings/` is getting being redirected to `https://appium.io/docs/en/2.0/guides/settings/`. Do you still want me to make the changes as suggested by you?

Reply:
Yes, it's supposed to redirect, I'm just thinking of the possibility when the docs eventually get updated to 2.1, 2.2, etc, in which case this link may become outdated. @jlipps what do you think?

Reply:
yes, ideally all our doc links should refer to `latest` so that the redirect gets to the most current version.